index.js

```diff
@@ -5,5 +5,5 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
```

Comment:
Unrelated, but from my debugging session, this assignment does not actually occur, as afterwards, `this.value` remains undefined. Or, actually, not defined at all.